### ISSUE TITLE
Installer poetry path

### ISSUE THREAD

**Issue by seanthegeek** [kevoreilly/CAPEv2#2435]
@doomedraven 

`installers/capev2.sh` currently installs Poetry in `/etc/poetry` which means that the `poetry` executable is installed in `/etc/poetry/bin/poetry`. The `/etc` path is generally used for configuration files alone, so it seems very weird to have Poetry installed there. `/opt/poetry` would be better. 

Many places in the installer assume that the `poetry` executable is located at `/etc/poetry/bin/poetry`. On my CAPE system that I set up years ago, the `poetry` executable is installed in a more normal location, `/usr/local/bin/poetry`. I have no idea how I originally installed it though.

Could the installer check various locations for the `poetry` executable, or at least allow the path to be passed as an argument?

**Comment by doomedraven** [comment]
Yes, I don't remember why, but you can see in PRs why, as far as I remember
APT's poetry is outdated or something like that. We could add it, I will
leave it open till I have some time to add it

**Comment by seanthegeek** [comment]
I just had a thought. For not, I'll just add a symlink on my system.

**Comment by doomedraven** [comment]
symlink worked?
